src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -34,7 +34,7 @@
 from azure.cli.command_modules.acs.mgmt_acs.lib \
     import AcsCreationClient as ACSClient
 # pylint: disable=too-few-public-methods,too-many-arguments,no-self-use,line-too-long
-from azure.cli.core._util import CLIError
+from azure.cli.core._util import CLIError, shell_safe_json_parse
```

Comment:
Just a thought. We should rename `_util` module to `util`. That fact that this module is imported from outside of the `azure.cli.core` module proves that it is not private module anymore. Leaving the name unchanged will cause ambiguity.